src/Signer/Key.php

```diff
@@ -83,7 +83,7 @@ public function getContent()
     }
 
     /**
-     * @return string
+     * @return null|string
```

Comment:
This contradicts the type of the property, which also needs to be adjusted

Reply:
OK so in this case I will change the default argument in the constructor.

Reply:
I think it’s preferable to set an empty string because other methods/functions expect to receive a string as argument (e.g. [OpenSSL::getPrivateKey](https://github.com/lcobucci/jwt/blob/3.4/src/Signer/OpenSSL.php#L44))